### DISCUSSION TITLE
Naive Questions:

### DISCUSSION THREAD

**Discussion by shersoni610** [crewAIInc/crewAI#1129] [Q&A]
Hello, 

Can someone answer some naive questions:

1.  Why agents have "backstory" and Task have "Description". Why do not we have only one? In the task, we provide agent then cann't it use Agent's backstory?
2.   Why we need to provide both agents and tasks to the Crew? We are providing an agent to the task then why don't we use it internally? Can it be potentially dangerous if the agents list and task list do not match?  Say in the example posted on the web, the first task is task1 in the tasks array and by mistake, in the agents in write [writer, researcher] then what could happen? 

Thanks

**Comment by Biancamazzi** [comment]
don't forget we have https://chat.openai.com/g/g-qqTuUWsBY-crewai-assistant to help and also the discord community https://discord.com/invite/X4JWnZnxPb

**Comment by C-1pher** [comment]
1. The backstory is the system prompt , while the description is the user's prompt ,so you need both of them to help the llm understand what to do exactly  .

2.The reason is that when you define the agents in the task  you just telling it what agents it should use for this exact task , while in the Crew you are telling it to load all the agents that it will need for the entire thing , so the Crew loads them and the task choses which one it needs from them .
